java/code/src/com/suse/manager/webui/websocket/RemoteMinionCommands.java

```diff
@@ -133,8 +135,10 @@ public void onMessage(Session session, String messageBody) {
             int timeOut = 600; // 10 minutes
 
             if (msg.isPreview()) {
+
                 List<String> allVisibleMinions = MinionServerFactory
                         .lookupVisibleToUser(webSession.getUser())
+                        .filter(m -> !mm.isSystemInMaintenanceMode(m))
```

Comment:
I think the ! (not) is wrong here. When a system is *in* maintenance mode, we can run remote commands.

Reply:
Agreed, we discussed offline and I think Martin is about to fix it.